docs/INITRD_INTERFACE.md

```diff
@@ -16,8 +16,8 @@ initrd and initrd-less boots. If an initrd is used, it is a good idea to pass a
 few bits of runtime information from the initrd to systemd in order to avoid
 duplicate work and to provide performance data to the administrator. In this
 page we attempt to roughly describe the interfaces that exist between the
-initrd and systemd. These interfaces are currently used by dracut and the
-ArchLinux initrds.
+initrd and systemd. These interfaces are currently used by mkosi-generated
```

Comment:
make mkosi a link to the project page? `[mkosi](https://github.com/systemd/mkosi)`